src/index.ts

```diff
@@ -142,7 +142,7 @@ export enum SigningAlgorithm {
  */
 export enum EncodingAlgorithm {
   Hex = "hex",
-  Base64 = "secp256k1",
+  Base64 = "base64",
```

Comment:
@stepan-romankov-fi I assume the Snap isn't using this SDK? Else I guess it wouldn't have worked